java/src/main/java/com/xxAMIDOxx/xxSTACKSxx/menu/mappers/RequestToCommandMapper.java

```diff
@@ -23,6 +25,10 @@ public static UpdateMenuCommand map(String correlationId, UUID menuId, UpdateMen
                 r.getDescription(), r.getEnabled());
     }
 
+    public static DeleteMenuCommand map(String correlationId, UUID menuId) {
```

Comment:
You might have issues with this later using the generic name and arguments. It's not actually mapping anything either?

Reply:
In fact it's completely unnecessary. Just instantiate the DeleteMenuCommand in the controller